frontend/interfaces/policy.ts

```diff
@@ -36,7 +36,7 @@ export interface IPolicy {
   author_email: string;
   resolution: string;
   platform: SelectedPlatformString;
-  team_id?: number;
+  team_id?: number | null;
```

Comment:
team_id null is shown in the new API https://github.com/fleetdm/fleet/pull/17227/files currently line 6074